### PR TITLE
Add option to set `silent` flag true or false on requests

### PR DESCRIPTION
**Pull request by pedrouid**

### CODE REVIEW

File: packages/types/index.d.ts [hunk 1]
```diff
@@ -258,4 +258,9 @@ declare module '@walletconnect/types' {
     rpc?: IRPCMap
     infuraId?: string
   }
+
+  export interface IRequestOptions {
+    _topic?: string
```
Comment (pedrouid) [inline packages/types/index.d.ts]: Remove custom topic option

File: packages/types/index.d.ts [hunk 2]
```diff
@@ -258,4 +258,9 @@ declare module '@walletconnect/types' {
...
+    pushNotification?: boolean
```
Comment (pedrouid) [inline packages/types/index.d.ts]: Rename pushNotification to forcePushNotification

File: packages/types/index.d.ts (hunk 1 above)
Comment (maricalucian) [reply packages/types/index.d.ts]: Moved topic back to separate param, and made options second optional param

File: packages/types/index.d.ts (hunk 2 above)
Comment (maricalucian) [reply packages/types/index.d.ts]: Done

**Comment by pedrouid** [comment]
Thanks @maricalucian, I've merged into a branch to tweak it before releasing. Check #230 for updates

**Pull request merged**